homework/Laura_Battaglia/Laura_Battaglia_datawarehouse_assignment.py

```diff
@@ -0,0 +1,60 @@
+import pandas as pd
+import numpy as np
+import dataset
+import datetime
+
+#read csv files as pandas dataframes
+
+employees = pd.read_csv('warehousing-operational-databases/homework/extracts/employees.csv')
+orders = pd.read_csv('warehousing-operational-databases/homework/extracts/orders.csv')
+products = pd.read_csv('warehousing-operational-databases/homework/extracts/products.csv')
+
+#define relational tables (and drop duplicates)
+offices = employees[['office_code', 'city','state', 'country','office_location']].copy()
+offices = offices.drop_duplicates()
+
+new_employees = employees[['employee_number', 'last_name', 'first_name','reports_to','job_title','office_code']].copy()
+new_employees = new_employees.drop_duplicates()
+
+#replace nas with -1 so to be able to change type to int
+new_employees.reports_to = new_employees.reports_to.fillna('-1').astype(int)
+
+
+customers = orders[['customer_number','customer_name','contact_last_name', 'contact_first_name', 'city', 'state', 'country', 'sales_rep_employee_number', 'credit_limit', 'customer_location']].copy()
+customers = customers.drop_duplicates()
+
+new_orders = orders[[ 'order_number', 'order_date', 'required_date' ,'shipped_date', 'status', 'comments', 'customer_number']].copy()
+new_orders = new_orders.drop_duplicates()
+
+#replace nas with 1111-11-11 so to be able to change type to date
```

Comment:
All these "custom null types" are a bit yucky for me. I would just do the casting in Python and replace with None's, then you are having everything proper in Python and THEN put it into the db. But that's a small thing.